org.kde.kpat.json

```diff
@@ -0,0 +1,115 @@
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/19.12.0/src/libkdegames-19.12.0.tar.xz",
+                    "sha256": "4498e462f248e44bbd7fe28e5c8a47d43cdd058e813c47e8fe228b8d2439ad1f"
+                }
+            ]
+        },
+        {
+            "name": "freecell-solver",
+            "buildsystem": "cmake",
+            "config-opts": [ "-DFCS_WITH_TEST_SUITE=OFF", "-DFCS_BUILD_DOCS=OFF" ],
+            "build-options" : {
+                "env": {
+                    "PERLLIB": "/app/lib/perl5/site_perl/5.30.0:/app/lib/perl5/site_perl/5.30.0/arch"
+                }
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://fc-solve.shlomifish.org/downloads/fc-solve/freecell-solver-5.14.0.tar.xz",
+                    "sha256": "b09e7394181fcd972436fbbcd2192a9085b8924b70c892ad88b9ad95a7abee12"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "random2",
+                    "buildsystem": "simple",
+                    "build-commands": [
+                        "pip3 install --prefix=/app --no-deps ."
+                    ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://files.pythonhosted.org/packages/ee/b5/5aca5217501fbd06b1e5aed6d5986baa0da5380880e6bb6f421ed18e3a32/random2-1.0.1.zip",
+                            "sha256": "34ad30aac341039872401595df9ab2c9dc36d0b7c077db1cea9ade430ed1c007"
+                        }
+                    ]
+                },
+                {
+                    "name": "pbr",
+                    "buildsystem": "simple",
+                    "build-commands": [
+                        "pip3 install --prefix=/app --no-deps ."
+                    ],
+                    "sources": [ { "type": "git", "url": "https://github.com/openstack/pbr.git", "branch": "5.4.4" } ]
+                },
+                {
+                    "name": "pysol_cards",
+                    "buildsystem": "simple",
+                    "build-commands": [
+                        "pip3 install --prefix=/app --no-deps ."
+                    ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://pypi.io/packages/source/p/pysol_cards/pysol_cards-0.8.7.tar.gz",
+                            "sha256": "b723c94ee5ccaa86ac40208cdd0b10e30f403f0c739731e5484e53a7932eecf8"
+                        }
+                    ]
+                },
+                {
+                    "name": "perl-modules",
+                    "buildsystem": "simple",
+                    "build-commands": [
+                        "echo 'this is a bit sad'",
```

Comment:
```suggestion
```
Not needed.

Reply:
Makes my life happier

Reply:
I mean you understand i obviously know the echo is not needed, right?